src/cosmos/authz/v1beta1/genesis.ts

```diff
@@ -1,44 +1,48 @@
+//@ts-nocheck
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
-import { GrantAuthorization } from "./authz";
```

Comment:
When I upgraded, somehow this import is now missing. I am not sure yet why because I don't see any changes to the protos that are major, but will get it sorted out.

Reply:
figured it out! Will take a moment to come up with the best solution. 

Now we have multiple versions of certain modules.

- [ ] should I just de-dup them and leave it up to the developer?
- [ ] maybe we print a warning and do a simple de-duplication

https://gist.github.com/pyramation/387059717e3db7ec1a24c2788a425cae

Reply:
ok fixed this in a mathematically identical style. 

We only need to do a single telescope process, but we order first to use cosmos-sdk then wasm.  Seems to be identical from checking all the files.